src/componets/TextGenerationForm.tsx

```diff
@@ -0,0 +1,184 @@
+"use client";
+
+import * as Switch from "@radix-ui/react-switch";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import "@/styles/switch.scss";
```

Comment:
please do some research on how to set up a`main.scss` and import other modules into it. Ideally we only need to import scss once in the `app/layout.tsx` .

Reply:
Gotcha! I will fix that today thank you so much!